layer/executables/layer_runtime.py

```diff
@@ -97,6 +104,50 @@ def _create_dataset(self, name: str, func: Callable[..., Any]) -> None:
         )
         data_catalog.store_dataset(func(), uuid.UUID(build_response.id.value))
 
+    def _create_model(self, name: str, func: Callable[..., Any]) -> None:
+        model = func()
+        model_flavor = get_flavor_for_model(model)
+        if model_flavor is None:
+            raise LayerFunctionRuntimeError("unsupported model flavor")
+        with tempfile.TemporaryDirectory() as model_dir:
+            model_flavor.save_model_to_directory(model, Path(model_dir))
+            model_catalog = ModelCatalogClient.create(self._client_config, self._logger)  # type: ignore
+            project = current_project_full_name()
+            asset_path = AssetPath(
+                name,
+                AssetType.MODEL,
+                org_name=project.account_name,  # type: ignore
+                project_name=project.project_name,  # type: ignore
+            )
+            source_digest = _get_function_source_digest(self._package_info.metadata)  # type: ignore
+            display_fabric = Fabric.F_LOCAL.value  # the fabric to display in the UI
```

Comment:
why is this `LOCAL` ? I guess it has to be a parameter we pass to the function. This is only for reporting purposes. The fabric should be 'consumed' by the executor service to create the right deployment.

Reply:
Due to legacy reasons, it has to be passed. I had to put any value here. We would need to pass it from the runtime. Same applies to the datasets. Will address in a separate PR.

Reply:
https://linear.app/layer/issue/LAY-3646/read-fabric-value-from-the-env-in-layer-runtime